kivy/uix/screenmanager.py

```diff
@@ -1013,11 +1013,13 @@ def remove_widget(self, *l):
         self.screens.remove(screen)
 
     def clear_widgets(self, screens=None):
-        if not screens:
+        if screens is None:
             screens = self.screens
-        remove_widget = self.remove_widget
```

Comment:
Just FYI, the reason for `remove_widget = self.remove_widget` is performance, but it probably doesn't really make a difference here either.